### DISCUSSION TITLE
Allow configuring "Add New" / "Select Existing" for M2O interface

### DISCUSSION THREAD

**Discussion by HemboPagi** [directus/directus#11903] [Feature Requests]
Hi, I would like to raise the topic as proposed at #6012.

I need users able to create (ideally also to delete without opening the related table) new content, but not to select from exiting list. 

My case is that I have create a table for date sequences with some filtering and logic. When clicking add new date (+) then date form is displayed with more fields. Those values should not be selectable as it does not make any sense.

Many thanks 
H

**Comment by rijkvanzanten** [comment]
The referenced discussion was implemented. You can create new items straight from the interface:

<img width="508" alt="CleanShot 2022-03-01 at 11 16 13@2x" src="https://user-images.githubusercontent.com/9141017/156206296-3102faa4-36fb-4d8e-9929-b9379a277b15.png">

**Reply by HemboPagi** [reply]
Thank you for your replay. My question was about disabling possibility to select an existing item (arrow down button):
<img width="508" alt="156206296-3102faa4-36fb-4d8e-9929-b9379a277b15" src="https://user-images.githubusercontent.com/95413906/156500865-3072d01e-f05f-4b5f-bf38-0de549f8b5bc.png">

**Reply by rijkvanzanten** [reply]
Ahh gotcha. Yes good one 👍🏻 

I'll mark this a feature request then 🙂

**Reply by HemboPagi** [reply]
👍🏻

**Reply by OzzyW80** [reply]
@rijkvanzanten hi is there a specific idea behind 2M buttons differs from 2O buttons? In a way the 2M buttons are inconsistent with the rest of the form. Not really critical thing but have thought about it several times.

**Reply by benhaynes** [reply]
@OzzyW80 — do you mean because the ones above are inline icons versus full buttons on the form? The inline icons allow the field to fit within a standard form better, so you can have that M2O dropdown beside an input and they fit nicely together. That's not really possible for O2M, since they are longer lists anyways. 

That being said, I'm all about consistency, so I'm open to discussing!

**Reply by HemboPagi** [reply]
Agree, they should look the same. As a user, I find it is a bit tricky to see the plus (+) icon next to the arrow, but i like the compactness.

**Reply by OzzyW80** [reply]
@benhaynes hmm, maybe I misunderstood you Ben but what I mean: the large buttons under a O2M field really break the consistency of the form UI as it stand out from all other fields. But as I think about it I realize now why they are there. When 10 items are added to that section they are useful. So just disregard my comment.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
